rotkehlchen/externalapis/github.py

```diff
@@ -18,7 +17,9 @@ def _query(self, path: str) -> dict:
         - RemoteError if there is a problem querying Github
         """
         try:
-            response = requests.get(url=f'{self.prefix}{path}', timeout=DEFAULT_TIMEOUT_TUPLE)
+            from rotkehlchen.db.settings import \
+                CachedSettings  # avoid circular import # pylint: disable=import-outside-toplevel
+            response = requests.get(url=f'{self.prefix}{path}', timeout=CachedSettings().get_timeout_tuple())  # noqa: E501
```

Comment:
This is not a good solution. Find a different way to do this. We already seem to do this in line 34, so something is wrong with this file.

What's more we never ever do what you did about splitting the line with `\`. Please don't do arbitrary stuff like this.

Generally these new CachedSettings should be in their own file with no other imports at all. To avoid all such import errors.

Reply:
ok, isort does this splitting automatically. I whould isort:skip this line

Reply:
I noticed that if you remove the "as" in the import and just add a long comment, isort will split the line like in my case. For example copy paste my import.

Reply:
we never ever do this. Adjust your editor to not do this. Also I think that if you move it to its own file as adviced then you can fix the circular import and remove this hack from here